lib/ansiblelint/__main__.py

```diff
@@ -118,7 +118,7 @@ def main():
 
         if 'parseable_severity' in config:
             options.parseable_severity = options.parseable_severity or \
-                                         config['parseable_severity']
+                config['parseable_severity']
```

Comment:
This reads weirdly, I'd prefer braces. But again, this has nothing to do with docstrings so should be submitted separately.